deployer/utils.py

```diff
@@ -83,12 +83,13 @@ def verify_and_decrypt_file(encrypted_path):
             decrypted contents. Unless the file is not valid JSON/YAML or does not have
             the prefix `enc-`, then we return the original, encrypted path.
     """
+    assert_file_exists(encrypted_path)
     filename = os.path.basename(encrypted_path)
     _, ext = os.path.splitext(filename)
 
     # Our convention is that encrypted secrets in the repository begin with "enc-",
     # so first we check for that
```

Comment:
```suggestion
    # Our convention is that encrypted secrets in the repository begin with "enc-" and include
    # "secret" in the filename, so first we check for that. We use an 'or' conditional here since
    # we want to catch files that contain "secret" but do not have the "enc-" prefix and ensure
    # they are encrypted, raising an error if not.
```